compiler/one-cmds/CMakeLists.txt

```diff
@@ -32,26 +32,26 @@ foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
           
 endforeach(ONE_COMMAND)
 
-set(ONE_COMMAND_FILES
+set(ONE_DOCUMENT_FILES
     how-to-use-one-commands.txt
     how-to-prepare-virtualenv.txt
 )
 
-foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
+foreach(ONE_DOCUMENT IN ITEMS ${ONE_DOCUMENT_FILES})
 
-  set(ONE_COMMAND_FILE ${ONE_COMMAND})
-  set(ONE_COMMAND_SRC "${CMAKE_CURRENT_SOURCE_DIR}/${ONE_COMMAND_FILE}")
-  set(ONE_COMMAND_BIN "${CMAKE_CURRENT_BINARY_DIR}/${ONE_COMMAND_FILE}")
-  set(ONE_COMMAND_TARGET "${ONE_COMMAND}_target")
+  set(ONE_DOCUMENT_FILE ${ONE_DOCUMENT})
+  set(ONE_DOCUMENT_SRC "${CMAKE_CURRENT_SOURCE_DIR}/${ONE_DOCUMENT_FILE}")
+  set(ONE_DOCUMENT_BIN "${CMAKE_CURRENT_BINARY_DIR}/${ONE_DOCUMENT_FILE}")
+  set(ONE_DOCUMENT_TARGET "${ONE_DOCUMENT}_target")
 
-  add_custom_command(OUTPUT ${ONE_COMMAND_BIN}
-    COMMAND ${CMAKE_COMMAND} -E copy "${ONE_COMMAND_SRC}" "${ONE_COMMAND_BIN}"
-    DEPENDS ${ONE_COMMAND_SRC}
-    COMMENT "Generate ${ONE_COMMAND_BIN}"
+  add_custom_command(OUTPUT ${ONE_DOCUMENT_BIN}
+    COMMAND ${CMAKE_COMMAND} -E copy "${ONE_DOCUMENT_SRC}" "${ONE_DOCUMENT_BIN}"
+    DEPENDS ${ONE_DOCUMENT_SRC}
+    COMMENT "Generate ${ONE_DOCUMENT_BIN}"
   )
 
-  add_custom_target(${ONE_COMMAND_TARGET} ALL DEPENDS ${ONE_COMMAND_BIN})
+  add_custom_target(${ONE_DOCUMENT_TARGET} ALL DEPENDS ${ONE_DOCUMENT_BIN})
 
-  install(FILES ${ONE_COMMAND} DESTINATION bin)
+  install(FILES ${ONE_DOCUMENT} DESTINATION doc)
```

Comment:
(Optoinal) I guess that you do not need to create a target with  ``add_custom_target`` ``)

> File names given as relative paths are interpreted with respect to the current source directory.

``install(FILES ${ONE_DOCUMENT} DESTINATION doc)`` would be enough ``)...